cvat/apps/dataset_manager/management/commands/exportcachecleanup.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2025 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import os
+import shutil
+from pathlib import Path
+
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from cvat.apps.engine.models import Job, Project, Task
+
+
+class Command(BaseCommand):
+    help = "Cleanup outdated export cache"
+
+    def handle(self, *args, **options):
+        def update_progress():
+            progress = (i + 1) / objects_count
+            done = int(progress_bar_len * progress)
+            progress_bar = "#" * done + "-" * (progress_bar_len - done)
+            self.stdout.write(f"\rProgress: |{progress_bar}| {progress:.0%}", ending="")
```

Comment:
This might not work on mac and windows. Why not use tqdm?

Reply:
Ask Roman about it

Reply:
@SpecLad, I saw you were saying it's not installed, but why not install it? Do you see any problems with that?

Reply:
> This might not work on mac and windows

I've checked this on mac and it works as expected. I also do not see the reason why it might not work on different terminals. I thought that carriage return should work similar on different operating systems/terminals.

> Frankly, I don't see a reason to get fancy with the output at all. If it was me, I would just print a message for every processed task/job/project and be done with it

I don't see the benefit of printing a message for every processed project/task/job when the number of objects is quite large. It's just a useless output.
So, if we are not going to add `tqdm` as a new dependency right now, I prefer to keep the current implementation.